src/app/common/validation/forms/address-validators.ts

```diff
@@ -33,6 +33,7 @@ export function btcAddressValidator() {
     });
 }
 
```

Comment:
Maybe we can just stop exporting this instead of disabling the checker?

Checking the code and it seems like we could even just remove `btcTaprootAddressValidator` altogether as it's not used anywhere now

Reply:
several times such disabling made by other developer saved me time, and I didn't write same code again, that's why normally I don't delete such functions

Reply:
OK makes sense. 

I usually favour deleting everything not in use as we spend much more time reading code than writing and un-used can clutter up the place. We can always recover it from GIT if needed